### PR TITLE
Contact us form

### LINKED ISSUE #171: Add Contact Us form
Fields should be: name, organization, work email (don't worry about validating this too much).

Add a contact us link to the top bar (rightmost link). Should redirect to a separate page which shows just the Contact Us form, and after it something like, "or if you prefer, Schedule a Demo"

### PR DESCRIPTION
**Pull request by t4lz**
Closes #171

Everything is copied from other pages, mostly from the waitlist form. ChatGPT helped me change it.
Didn't try to design it a lot, let me know if there is a different design you want for the form.

Todo: setup a backend endpoint for the form.

### CODE REVIEW

**Comment by t4lz** [comment]
Marked as ready to get a cloudflare preview link, but this is a WIP.

File: assets/css/site.css [hunk 1]
```diff
@@ -4,7 +4,7 @@ Version:		1.0.0
 Last change: 	03/06/2022
 -------------------------------------------------------------------*/
 
-/*! tailwindcss v3.2.4 | MIT License | https://tailwindcss.com*/
```
Comment (eyalb181) [inline assets/css/site.css:7]: I'm not sure you're supposed to commit this, maybe try to revert and see if the cloudflare still looks okay.

File: layouts/_default/contact.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+              class="contact-us__input mb-3 w-full outline-none focus:outline-none text-base text-[#222222] placeholder:text-[#222222] focus-within:text-[#222222] placeholder:font-light border border-solid border-[#222222] rounded-[5px] lg:rounded-[10px] px-8 py-[10px] lg:py-4" id="message" name="message"
+                      rows="4" placeholder="Your Message" data-required="true" required=""></textarea>
+            <div>
+              <button class="w-full h-[46px] lg:h-[52px] flex items-center justify-center text-primary bg-white border border-solid border-[#222222] rounded-[6px] lg:rounded-[10px] shadow-[-7px_6px_0px_#000000] lg:shadow-[-9px_8px_0px_#000000] transition-colors duration-200 ease-[ease]"
+                aria-label="Submit" form="contact-form" type="submit">
+                Submit {{ partial (printf "svg/icon-%s.svg" "waitlist") (dict "class" "w-6 h-6 text-current") }}
+              </button>
+            </div>
+          </form>
+          <div x-bind:class="(submitedMessage != null ? '' : 'scale-y-0') + ' transition py-3 text-center origin-top'">
```
Comment (eyalb181) [inline layouts/_default/contact.html:35]:
```suggestion
          <div x-bind:class="(submittedMessage != null ? '' : 'scale-y-0') + ' transition py-3 text-center origin-top'">
```

File: layouts/_default/contact.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                   placeholder="Work Email" data-required="true" required="">
+            <p class="mb-1">Message</p>
+            <textarea
+              class="contact-us__input mb-3 w-full outline-none focus:outline-none text-base text-[#222222] placeholder:text-[#222222] focus-within:text-[#222222] placeholder:font-light border border-solid border-[#222222] rounded-[5px] lg:rounded-[10px] px-8 py-[10px] lg:py-4" id="message" name="message"
+                      rows="4" placeholder="Your Message" data-required="true" required=""></textarea>
+            <div>
+              <button class="w-full h-[46px] lg:h-[52px] flex items-center justify-center text-primary bg-white border border-solid border-[#222222] rounded-[6px] lg:rounded-[10px] shadow-[-7px_6px_0px_#000000] lg:shadow-[-9px_8px_0px_#000000] transition-colors duration-200 ease-[ease]"
+                aria-label="Submit" form="contact-form" type="submit">
+                Submit {{ partial (printf "svg/icon-%s.svg" "waitlist") (dict "class" "w-6 h-6 text-current") }}
```
Comment (eyalb181) [inline layouts/_default/contact.html:31]: Doesn't look great with the icon - try spacing them out, or removing either the text or the icon, and seeing what looks best

File: config/_default/menus/menus.yaml [hunk 4]
```diff
@@ -24,6 +24,10 @@ main:
   name: Support
   url: "/#get-support"
   weight: 70
+- identifier: contact
```
Comment (eyalb181) [inline config/_default/menus/menus.yaml:27]: Please add to the footer as well

File: layouts/_default/contact.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+          <form id="contact-form" method="post" action="" x-on:submit="submitedMessage = await handleFormSubmit(event)">
+            <p class="mb-1">Name</p>
+            <input
+              class="contact-us__input mb-3 w-full outline-none focus:outline-none text-base text-[#222222] placeholder:text-[#222222] focus-within:text-[#222222] placeholder:font-light border border-solid border-[#222222] rounded-[5px] lg:rounded-[10px] px-8 py-[10px] lg:py-4" type="text" id="name" name="name"
+                   placeholder="Your Name" data-required="true" required="">
+            <p class="mb-1">Organization</p>
+            <input
+              class="contact-us__input mb-3 w-full outline-none focus:outline-none text-base text-[#222222] placeholder:text-[#222222] focus-within:text-[#222222] placeholder:font-light border border-solid border-[#222222] rounded-[5px] lg:rounded-[10px] px-8 py-[10px] lg:py-4" type="text" id="organization" name="organization"
+                   autocomplete="off" placeholder="Organization">
+            <p class="mb-1">Email</p>
```
Comment (eyalb181) [inline layouts/_default/contact.html:20]: It's weird that the fields have both a label and a placeholder - I'd leave just the label (and change Email to Work Email)

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: layouts/_default/contact.html (hunk 2 above)
Comment (t4lz) [reply layouts/_default/contact.html]: done https://github.com/metalbear-co/mirrord.dev/pull/172/commits/b94f46d69617fa25e16812a9a0a539ed121db9ca

File: layouts/_default/contact.html (hunk 3 above)
Comment (t4lz) [reply layouts/_default/contact.html]: Added spacing in https://github.com/metalbear-co/mirrord.dev/pull/172/commits/255b1fc010d443fbb714524b96fc76f84eadf421, is it better now?

File: config/_default/menus/menus.yaml (hunk 4 above)
Comment (t4lz) [reply config/_default/menus/menus.yaml:27]: Done in https://github.com/metalbear-co/mirrord.dev/pull/172/commits/e7d3dbc2e54020c718ae9d949ec05c7a7cd7f392

File: layouts/_default/contact.html (hunk 5 above)
Comment (t4lz) [reply layouts/_default/contact.html]: Done in https://github.com/metalbear-co/mirrord.dev/pull/172/commits/863fe1cf22778676aabea54a2cac3d09cac1b879

File: layouts/_default/contact.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    <div class="container w-full max-w-screen-lg mx-auto px-4 relative before:absolute before:content-[''] before:top-[50%] before:-left-[30%] before:w-[200%] before:bg-white before:h-[50%]">
+      <div class="bg-tertiary rounded-3xl py-6 px-2 relative">
+        <header class="text-center">
+          <h1>Contact Us</h1>
+        </header>
+
+        <div class="contact-us__form w-10/12 lg:w-1/2 mx-auto relative" x-data="{ submittedMessage: null }">
+          <form id="contact-form" method="post" action="https://api.web3forms.com/submit">
+            <input type="hidden" name="access_key" value="39afe569-931e-42d7-b2c7-97e43cde610b">
+            <label for="name">Your Name</label>
```
Comment (eyalb181) [inline layouts/_default/contact.html:13]:
```suggestion
            <label for="name">Name</label>
```

File: yarn.lock
Comment (t4lz) [inline yarn.lock:1]: Is this ok to commit or should I try and see if now cloudflare preview works without these changes (it didn't initially but maybe that was unrelated).

Comment (eyalb181) [reply yarn.lock:1]: Lets try without

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: layouts/_default/contact.html [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+          form.reset();
+          setTimeout(() => {
+            result.style.display = "none";
+          }, 15000);
+        });
+    });
+  </script>
+
+
+  {{- partial "form/handle-submission.html" (dict "context" . "prefix" "contact" "successMessage" "Thank you for contacting us! We'll get back to you soon." "failureMessage" "Failed to send your message. Please contact us at support@metalbear.co.") }}
```
Comment (eyalb181) [inline layouts/_default/contact.html:95]: Is this still necessary? Thought it was handled by web3forms?

Comment (t4lz) [reply layouts/_default/contact.html:95]: You're right.

File: yarn.lock
Comment (t4lz) [reply yarn.lock:1]: Looks like it's working without.

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**